test/selenium_helpers.js

```diff
@@ -34,25 +31,163 @@ const setupSelenium = function(done) {
       })
     }
 
-  this.getLogs = () => {
-    return this.browser.manage().logs().get('browser')
+  browser.getLogs = function(){
+    return this.manage().logs().get('browser')
       .then(logs => console.log('Browser Logs:', logs))
   }
 
-  this.loginAs = loginAs
-  done()
+  browser.loginAs = function(githubId) {
+    return this.visit(`/__login/${githubId}`)
+    .then(session => {
+      const body = this.findElement(By.css('body'), 2000)
+      return this.wait(
+        until.elementTextContains(body, `logged in as ${githubId}`),
+        5000
+      )
+    })
+  }
+
+  browser.closeTabs = function(){
+    return this.getAllWindowHandles()
+      .then(handles => {
+        for(let i = handles.length -1; i > 0; i--){
```

Comment:
it took me a second to realize this loop doesnt loop over index zero. I'd prefer something more explicit

```js
handles.forEach((handle, index) => {
  if (index === 0) return
  this.switchTo().window(handle)
  this.close()
})
```